Directory.Build.props

```diff
@@ -3,7 +3,7 @@
     <DotnetVersion>net6.0</DotnetVersion>
     <AssemblyVersion>2.0.0.0</AssemblyVersion>
     <FileVersion>2.0.0.0</FileVersion>
-    <Version>2.106.4</Version>
+    <Version>2.106.6-preview5</Version>
```

Comment:
I'd say to avoid preview suffixes since this is managed by us

Reply:
yes, this was because I had to teste it on multiple platforms. Win and Mac, so created the suffix.
The final version wont have it